app/src/main/java/net/pp3345/ykdroid/yubikey/UsbYubiKey.java

```diff
@@ -96,6 +109,20 @@ public String getName() {
 		public String getVersion() {
 			return this.version;
 		}
+
+		public static Type lookupDeviceType(final UsbDevice device) {
+			for (final Type type : Type.values()) {
+				if (type.getVendorID() == device.getVendorId() &&
+						type.getProductID() == device.getProductId()) {
+					return type;
+				}
+			}
+			return Type.UNKNOWN;
```

Comment:
Please add a blank line before this one.

Reply:
Done